central/sensor/service/pipeline/nodescansv2/pipeline.go

```diff
@@ -49,7 +49,7 @@ func (p *pipelineImpl) Run(ctx context.Context, clusterID string, msg *central.M
 	}
 
 	// TODO(ROX-12240, ROX-13053): Do something meaningful with the nodeInventory
-	log.Infof("Central received NodeInventory: %+v", nodeInventory)
+	log.Debugf("Central received NodeInventory: %+v", nodeInventory)
```

Comment:
I see you already incorporated the feedback regarding loglevels. Great job!

That said, we're keeping the pipeline only until #3757 is merged, as `NodeInventory` is still an independent message here, and we depend on this pipeline to print the inventory, correct?

Reply:
Yes, no changes to the pipeline here as it affects the decision that we need to do in #3757.

The nodescanv2 pipeline is a dud and does nothing except of printing the debug log message. Moreover, it is guarded by the feature-flag - see [here](https://github.com/stackrox/stackrox/blob/b5394616a49b9999da63534840522c80d9277677/central/sensor/service/pipeline/all/factory.go#L68-L70)